docs/docs/tutorial/chapter4/authentication.md

```diff
@@ -849,22 +840,16 @@ export const requireAuth = ({ roles }: { roles?: AllowedRoles } = {}) => {
   }
 }
 ```
+</TabItem>
+</Tabs>
 
-:::caution
-
-At this point of the tutorial we have **not added roles** to our user model yet, therefore you can ignore the following error:
-
-`Property 'roles' does not exist on type '{ id: number; email: string; }'.`
+:::info
 
-in the `hasRole` method in `api/src/lib/auth.ts` for now.
+At this point of the tutorial we have **not added roles** to our user model yet, therefore you can ignore the `hasRole` method in `api/src/lib/auth.js` for now.
```

Comment:
```suggestion
At this point of the tutorial we have **not added roles** to our user model yet, therefore you can ignore the `hasRole` method in `api/src/lib/auth.{js,ts}` for now.
```

Actually, thinking about it, can't we just remove this whole info-block now then? I think we added it just to explain the error we used to get

Reply:
Fine with me. The info box however never said anything about an "error" – i thought it may be helpful to leave it in just to make users curious about the RBAC-chapter. But as that's clearly visible in the navigation, people can easily deduct themselves that where to go for that i guess.

So i'm not sure if there ever would have been an error on the js side of things – but on the other hand i'll probably never be able to tell as i've been on the TS side of things from day one :wink:

Reply:
It did explicitly mention an error in the TS version of the info box. And for the JS version it kind of implicitly did by mentioning "missing field". (Which the note still does).

So yeah, I think we should just remove this box now :)

Reply:
08fac71d72a2e15a5bd70e7254a64bb11510423e

Reply:
![image](https://user-images.githubusercontent.com/30793/183325741-c510bd38-5099-4d7d-94c8-767ef423b22e.png)

It looks like it's still there. That commit you point to only changes the file extension

Reply:
Sorry, i simply clicked on your suggested change, committed that and called it a day  ;)

![grafik](https://user-images.githubusercontent.com/1634615/183326147-ebaa5ca0-0310-4cea-8653-cde6907ed54f.png)

But that was something else of course ;) Oh my, time for a nap. Here you go → 2e3507680